build.gradle

```diff
@@ -23,7 +22,7 @@ plugins {
 	id 'maven-publish'
 }
 
-def minecraft_version = "19w41a"
+def minecraft_version = "19w44a"
```

Comment:
Not sure why this is showing as a change?

Reply:
@natanfudge Rebase you branch against 19w44a.